modules/mediawiki/files/favicon.php

```diff
@@ -30,8 +30,17 @@ function streamFavicon() {
 
 	$status = $client->execute();
 	if ( !$status->isOK() ) {
-		header( 'HTTP/1.1 500 Internal Server Error' );
-		return;
+		$favicon = '/favicons/default.ico';
+		$url = wfExpandUrl( $favicon, PROTO_CANONICAL );
+		$client = MediaWikiServices::getInstance()
```

Comment:
Could we use a more specific name here & for $status?

Reply:
We don't need to. It's already used elsewhere, and this allows the rest to be executed. Just changing here won't work, and I want to maintain consistency.